src/RequestForm.js

```diff
@@ -689,7 +697,7 @@ class RequestForm extends React.Component {
                                     ref={this.itemBarcodeRef}
                                     onInput={this.onItemClickDebounce}
                                     onKeyDown={e => this.onKeyDown(e, 'item')}
-                                    validate={this.requireItem}
+
```

Comment:
If validation isn't needed on this field anymore, then the requireItem function can be removed altogether (and maybe the translation string that it uses).

Reply:
Good idea @Baroquem! I will keep the translation around because it's used in couple different places.